src/System/Router/renderServerApp.tsx

```diff
@@ -78,5 +82,39 @@ export const renderServerApp = ({
 
   const statusCode = getENV("statusCode") ?? code
 
-  res.status(statusCode).render(`${PUBLIC_DIR}/html.ejs`, options)
+  res
```

Comment:
non-blocker: do you think switch(true) would be better to read it here? It might be imo

Reply:
This works with express's funky callback style, i think. We'll have to revisit this when we get to the React 18 upgrade as this form of streaming is no longer supported. So this is all likely just temporary.

Reply:
What are some things to watch out for in production, especially metrics-wise, as a result of this once it lands and is enabled? Is it that, we should see maybe no change or only a very slight improvement on more modern devices w/ snappy connections, but a larger improvement on the lower powered ones?

Reply:
Basically TTFB, total blocking time, ICP -- anything that measures how fast something hits the user visually.

Reply:
The only way we can make this ultimately worthwhile would be to refactor our router in such a way as to ship the layout right away -- and then fetch and render subcontent. In effect, enable waterfall behavior. Not sure if its worth it to satisfy the bots when, at least on desktop, most of our pages on google search console are 🟢.